src/workerd/jsg/ser.c++

```diff
@@ -100,8 +100,7 @@ v8::Maybe<bool> Serializer::WriteHostObject(v8::Isolate* isolate, v8::Local<v8::
     jsg::Lock& js = jsg::Lock::from(isolate);
 
     if (object->InternalFieldCount() != Wrappable::INTERNAL_FIELD_COUNT ||
-        object->GetAlignedPointerFromInternalField(Wrappable::WRAPPABLE_TAG_FIELD_INDEX) !=
-            &Wrappable::WRAPPABLE_TAG) {
+        !object->IsApiWrapper()) {
```

Comment:
Hmm, this is only equivalent to the old code if _all_ API objects are `Wrappable` instances, that is, JSG is the only one creating API wrappers. But I'm not sure if this is true -- I think V8 itself might implement some types as API wrappers internally? Even if not, I don't want to preclude being able to bring in other libraries that implement V8 APIs in their own way.

Can we somehow check not just that it's a wrapper, but that it's _our_ wrapper? Probably by checking the tag?

Reply:
We might have to re-introduce an API api for that as the tag is not exposed in a way that is queryable. Also, the tag is now actually from the `CppHeapPointerTag` enum class which can also used internally by v8.

Reply:
Perhaps we need to continue to have two internal pointer fields, one of which continues to work like `WRAPPABLE_TAG`, but we only use it to identify our own objects...

Reply:
Done